### PR TITLE
[ACM-2.10] ACM#10464: Extracting OpenShift release image digest

### PR DESCRIPTION
**Pull request by xenolinux**
https://issues.redhat.com/browse/ACM-10464

The peer-review approval is present in https://github.com/stolostron/rhacm-docs/pull/6392

### CODE REVIEW

File: clusters/hosted_control_planes/create_cluster_bm.adoc [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
 
 +
 ----
-oc create ns <hosted-cluster-namespace>-<hosted-cluster-name>
+oc create ns <hosted_cluster_namespace>-<hosted_cluster_name>
 ----
 
 +
-Replace `<hosted-cluster-namespace>` with your hosted cluster namespace name, for example, `clusters`. Replace `<hosted-cluster-name>` with your hosted cluster name.
+Replace `<hosted_cluster_namespace>` with your hosted cluster namespace name, for example, `clusters`. Replace `<hosted-cluster-name>` with your hosted cluster name.
```
Comment (CrystalChun) [inline clusters/hosted_control_planes/create_cluster_bm.adoc]:
```suggestion
Replace `<hosted_cluster_namespace>` with your hosted cluster namespace name, for example, `clusters`. Replace `<hosted_cluster_name>` with your hosted cluster name.
```

**Review by CrystalChun — APPROVED** [review]
/lgtm

File: clusters/hosted_control_planes/create_cluster_bm.adoc [hunk 2]
```diff
@@ -1,51 +1,51 @@
 [#creating-a-hosted-cluster-bm]
 = Creating a hosted cluster on bare metal
 
-You can create a hosted cluster or import one. For instructions to import a hosted cluster, see _Importing a hosted cluster_.
+You can create a hosted cluster on bare metal or import one. For instructions to import a hosted cluster, see _Importing a hosted cluster_.
 
 . Create the hosted control plane namespace by entering the following command:
 
 +
 ----
```
Comment (dockerymick) [inline clusters/hosted_control_planes/create_cluster_bm.adoc:10]:
```suggestion
[source,bash]
----
```

File: clusters/hosted_control_planes/create_cluster_bm.adoc [hunk 3]
```diff
@@ -1,51 +1,51 @@
...
 
 +
-Replace `<hosted-cluster-namespace>` with your hosted cluster namespace name, for example, `clusters`. Replace `<hosted-cluster-name>` with your hosted cluster name.
+Replace `<hosted_cluster_namespace>` with your hosted cluster namespace name, for example, `clusters`. Replace `<hosted_cluster_name>` with your hosted cluster name.
 
 . Verify that you have a default storage class configured for your cluster. Otherwise, you might see pending PVCs. See the following example:
 
 +
 ----
```
Comment (dockerymick) [inline clusters/hosted_control_planes/create_cluster_bm.adoc:21]: similar question from my other review: https://github.com/stolostron/rhacm-docs/pull/6394#discussion_r1591192196

File: clusters/hosted_control_planes/create_cluster_bm.adoc [hunk 4]
```diff
@@ -1,51 +1,51 @@
...
 <6> Specify the path to your SSH public key. The default file path is `~/.ssh/id_rsa.pub`.
 <7> Specify your hosted cluster namespace.
-<8> Specify the supported {ocp-short} version that you want to use, for example, `4.14.0-x86_64`.
+<8> Specify the supported {ocp-short} version that you want to use, for example, `4.14.0-x86_64`. If you are using a disconnected environment, replace `<ocp_release_image>` with the digest image. To extract the {ocp-short} release image digest, see _Extracting the {ocp-short} release image digest_.
 
 +
 . After a few moments, verify that your hosted control plane pods are up and running by entering the following command:
 
 +
 ----
```
Comment (dockerymick) [inline clusters/hosted_control_planes/create_cluster_bm.adoc:50]:
```suggestion
[source,bash]
----
```

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: clusters/hosted_control_planes/create_cluster_bm.adoc (hunk 3 above)
Comment (xenolinux) [reply clusters/hosted_control_planes/create_cluster_bm.adoc:21]: This is a continuous one command. Added [source,bash] for this command too.

**Pull request merged**